README.md

```diff
@@ -2,112 +2,141 @@
 
 The TickTickSync plugin automatically creates tasks in ticktick and synchronizes task state between Obsidian and ticktick.
 
-The plugin works best when the [Tasks](https://github.com/obsidian-tasks-group/obsidian-tasks) plugin is installed. 
+The plugin works best when the [Tasks](https://github.com/obsidian-tasks-group/obsidian-tasks) plugin is installed.
+
+## Features
 
-## Features 
 - Any task created in Obsidian and taged with #ticktick will be synchronized with TickTick
 - Any task created in TickTick will be synchronized to Obisidian.
 - Updates are bi-directional
-- Tasks added or updated from Obsidian are synched to TickTick immediately. 
+- Tasks added or updated from Obsidian are synched to TickTick immediately.
 - Tasks added or updated from TickTick are synched on a configurable time interval.
 
 ## Notes
-1. TickTick lists are referred to as Projects in the plugin, and throughout documentation.
-2. TickTick only has four priorities (High, Medium, Low, None). Please see Priority in [Task Format](#Task-format) for Task mapping of priority
-3. Please see known issue 3. below. __Strongly__ recommend using subtasks rather than items to avoid loss of data.
 
+1. **All Task Deletion operations must be confirmed.** In the fullness of time, when we get out of Beta, this might be a preference item. For now it's compulsory.
+2. TickTick lists are referred to as Projects in the plugin, and throughout documentation.
+3. TickTick only has four priorities (High, Medium, Low, None). Please see Priority in [Task Format](#Task-format) for Task mapping of priority
+4. **Warning:** TickTickSync automatically creates .md files corresponding with the lists/projects created in TickTick and downloads all tasks to them.  
+   If the files are deleted in Obsidian while the TickTickSync plugin is active, those tasks will be deleted. If you accidentally confirm that deletion: 
```

Comment:
@thesamim Can you please make this path configurable. Where a user can specify a folder to store these .md files.

Reply:
On the roadmap: https://github.com/thesamim/TickTickSync/issues/12